syntax = "proto3";

package f1_predict.constants;

enum Circuit {
  CIRCUIT_UNSPECIFIED = 0;
  AUSTRALIA_CIRCUIT = 1;
  CHINA_CIRCUIT = 2;
  JAPAN_CIRCUIT = 3;
  BAHRAIN_CIRCUIT = 4;
  SAUDI_ARABIA_CIRCUIT = 5;
  MIAMI_CIRCUIT = 6;
  EMILIA_ROMAGNA_CIRCUIT = 7;
  MONACO_CIRCUIT = 8;
  SPAIN_CIRCUIT = 9;
  CANADA_CIRCUIT = 10;
  AUSTRIA_CIRCUIT = 11;
  GREAT_BRITAIN_CIRCUIT = 12;
  BELGIUM_CIRCUIT = 13;
  HUNGARY_CIRCUIT = 14;
  NETHERLANDS_CIRCUIT = 15;
  ITALY_CIRCUIT = 16;
  AZERBAIJAN_CIRCUIT = 17;
  SINGAPORE_CIRCUIT = 18;
  UNITED_STATES_CIRCUIT = 19;
  MEXICO_CIRCUIT = 20;
  BRAZIL_CIRCUIT = 21;
  LAS_VEGAS_CIRCUIT = 22;
  QATAR_CIRCUIT = 23;
  ABU_DHABI_CIRCUIT = 24;
  FRANCE_CIRCUIT = 25;
  PORTUGAL_CIRCUIT = 26;
  STYRIA_CIRCUIT = 27;
  RUSSIA_CIRCUIT = 28;
  TURKIYE_CIRCUIT = 29;
}

enum Team {
  TEAM_UNSPECIFIED = 0;
  MCLAREN = 1;
  FERRARI = 2;
  RED_BULL_RACING = 3;
  MERCEDES = 4;
  ASTON_MARTIN = 5;
  ALPINE = 6;
  HAAS = 7;
  RB = 8;
  WILLIAMS = 9;
  KICK_SAUBER = 10;
}

enum Driver {
  DRIVER_UNSPECIFIED = 0;
  ALEXANDER_ALBON = 1;
  CARLOS_SAINZ = 2;
  CHARLES_LECLERC = 3;
  ESTEBAN_OCON = 4;
  FERNANDO_ALONSO = 5;
  GABRIEL_BORTOLETO = 6;
  GEORGE_RUSSELL = 7;
  ISACK_HADJAR = 8;
  JACK_DOOHAN = 9;
  KIMI_ANTONELLI = 10;
  LANCE_STROLL = 11;
  LANDO_NORRIS = 12;
  LEWIS_HAMILTON = 13;
  LIAM_LAWSON = 14;
  MAX_VERSTAPPEN = 15;
  NICO_HULKENBERG = 16;
  OLIVER_BEARMAN = 17;
  OSCAR_PIASTRI = 18;
  PIERRE_GASLY = 19;
  YUKI_TSUNODA = 20;
  FRANCO_COLAPINTO = 21;
  VALTTERI_BOTTAS = 22;
  SERGIO_PEREZ = 23;
  KEVIN_MAGNUSSEN = 24;
  GUANYU_ZHOU = 25;
  DANIEL_RICCIARDO = 26;
  LOGAN_SARGEANT = 27;
  NYCK_DE_VRIES = 28;
  SEBASTIAN_VETTEL = 29;
  MICK_SCHUMACHER = 30;
  NICHOLAS_LATIFI = 31;
  ANTONIO_GIOVINAZZI = 32;
  KIMI_RAIKKONEN = 33;
  NIKITA_MAZEPIN = 34;
  ROBERT_KUBICA = 35;
}
